### ISSUE TITLE
Dialog not visible in FF

### ISSUE THREAD

**Issue by robrez** [PolymerElements/paper-dialog#47]
On firefox (41.0.1, windows7) --

If you place a paper-dialog inside a non-top-level element, the dialog is never visible.
eg:

```
<body>
  <x-element>
    <x-another-element>
      <paper-dialog/>
    </x-another-element>
  </x-element>
</body>
```

I will close this if it is a duplicate of #15 or if it belongs over in `paper-dialog-behavior`. I plan on testing the issue whenever a custom implementation of paper-dialog-behavior is used.

(modal dialog example from another repository).. https://github.com/robrez/poly-theme-builder/issues/3 -
![ffbug](https://cloud.githubusercontent.com/assets/1993389/10255826/99296980-6912-11e5-831c-af5bd776014d.jpg)

I seem to remember reading some documentation somewhere that stated that the dialog element should always be made a child of `<body>` whenever possible. I worked around that other issue by moving the dialog up a level and firing an event to request that the dialog be toggled.

_edit:_ I could make the dialog visible iff absolute positioning was used via a css rule.

**Comment by robrez** [comment]
_update_

If I use `paper-dialog-behavior/demo/simple-dialog.html` instead of `paper-dialog`, the dialog works as expected.

[simple-dialog.html](https://github.com/PolymerElements/paper-dialog-behavior/blob/master/demo/simple-dialog.html)

**Comment by valdrinkoshi** [comment]
This seems to be solved with `v1.0.3` http://jsbin.com/tadeje/1/edit?html,output

**Comment by robrez** [comment]
@valdrinkoshi thanks, I'll check it out
